gcp_variant_transforms/pipeline_common_test.py

```diff
@@ -178,26 +173,19 @@ def _create_mock_args(self, **args):
   def _get_pipeline_mode(self, args):
     all_patterns = pipeline_common._get_all_patterns(args.input_pattern,
                                                      args.input_file)
-    return pipeline_common.get_pipeline_mode(all_patterns,
-                                             args.optimize_for_large_inputs)
+    return pipeline_common.get_pipeline_mode(all_patterns)
 
   def test_get_mode_optimize_set(self):
```

Comment:
Same here, this test should be unnecessary.

Reply:
Done.